src/main/java/net/cytonic/cytosis/commands/TimeCommand.java

```diff
@@ -0,0 +1,77 @@
+package net.cytonic.cytosis.commands;
+
+import net.cytonic.cytosis.Cytosis;
+import net.minestom.server.command.builder.Command;
+import net.minestom.server.command.builder.arguments.ArgumentType;
+import net.minestom.server.command.builder.suggestion.SuggestionEntry;
+import static net.cytonic.cytosis.utils.MiniMessageTemplate.MM;
+
+/**
+ * The class representing the time command
+ */
+public class TimeCommand extends Command {
+
+    /**
+     * Creates a new command and sets up the consumers and execution logic
+     */
+    public TimeCommand() {
+        super("time");
+        setCondition((sender, _) -> sender.hasPermission("cytosis.commands.time"));
+        var timeArgument = ArgumentType.Word("time").from("day", "night", "noon", "midnight", "sunrise", "sunset");
+        var timeInteger = ArgumentType.Integer("timeInteger");
+        timeArgument.setSuggestionCallback((sender, _, suggestion) -> {
+            if (sender.hasPermission("cytonic.commands.time")) {
+                suggestion.addEntry(new SuggestionEntry("day"));
+                suggestion.addEntry(new SuggestionEntry("night"));
+                suggestion.addEntry(new SuggestionEntry("noon"));
+                suggestion.addEntry(new SuggestionEntry("midnight"));
+                suggestion.addEntry(new SuggestionEntry("sunrise"));
+                suggestion.addEntry(new SuggestionEntry("sunset"));
+            }
+        });
+        setDefaultExecutor((sender, _) -> {
+            if (sender.hasPermission("cytosis.commands.time")) {
+                sender.sendMessage(MM."<RED>Usage: /time (time)");
+            }
+        });
+        addSyntax((sender, context) -> {
+            if (sender.hasPermission("cytosis.commands.time")) {
+                switch (context.get(timeArgument).toLowerCase()) {
```

Comment:
This is not the right approach. You should add another method call to addSyntax so each one only has one argument. This will throw a null pointer exception if you supply an integer.

Reply:
Ok my bad.